### DISCUSSION TITLE
OOM error in CUDA

### DISCUSSION THREAD

**Discussion by SHAI-shamshad** [h2oai/h2o-llmstudio#574] [Q&A]
I have created an experiment with nvidia gpu machine with 24 GB of memory and the experiment ran successfully. Now when I am trying to run that same experiment(and keeping every config same as previous experiment). I am getting OOM. I am using mistral-7b-v0.1 base model. Any solution for that?

**Comment by psinger** [comment]
Same data? Does it oom each run?

Pytorch also sometimes has a tiny gpu leak of ~1GB, so stopping and restarting LLM Studio might reset that.

**Reply by SHAI-shamshad** [reply]
yes it OOM at each run. Will try restarting the LLM Studio.

**Reply by MananpreetKaur1** [reply]
Did that help?
